src/llm/bielik.py

```diff
@@ -0,0 +1,28 @@
+from langchain.chains.llm import LLMChain
+from langchain_ollama.llms import OllamaLLM
+from langchain_huggingface import HuggingFaceEmbeddings
+from langchain_core.prompts import PromptTemplate
+from langchain_community.vectorstores import FAISS
+
+
+class LLM:
+    def __init__(self, llm__model_name: str, template: str, embedding_model: HuggingFaceEmbeddings) -> None:
+        self.embedding_model = embedding_model
+        self.llm_model = OllamaLLM(model=llm__model_name)
+        self.prompt = PromptTemplate(input_variables=["question", "context"], template=template)
+        self.llm_chain = LLMChain(prompt=self.prompt, llm=self.llm_model, verbose=True)
+        self.question = "Co tam?"
+        self.vector_db = None
+
+    def load_vector_db(self, vector_db_path: str) -> None:
+        self.vector_db = FAISS.load_local(vector_db_path, self.embedding_model, allow_dangerous_deserialization=True)
+
+    def similarity(self) -> str:

```

Comment:
Maybe we can rename this function to get_similar_docs because similarity can be misleading. We can also make k as method argument and return list of all relevant docs, not only 1.